### PR TITLE
[10.0][ADD] product_packaging_barcode: Reimplement barcode in packagings

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: product_packaging_barcode/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Just use it
```
Comment (pedrobaeza) [inline product_packaging_barcode/readme/USAGE.rst]: Please which menu do you need to go to put the barcode

File: product_packaging_barcode/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Product Packaging Barcode',
+    'description': """
+        Reimplement barcode field in Product Packagings (was deleted in v9)""",
```
Comment (pedrobaeza) [inline product_packaging_barcode/__manifest__.py]: Remove this description, as it's taken from README.rst. Maybe you wanted to put `summary`?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Put also in ROADMAP something like:

* Add support for scanning the barcode on sales order screen for specifying the package.
* Add support for scanning the barcode on picking screen.

**Comment by rousseldenis** [comment]
@pedrobaeza The ROADMAP you propose is out of scope of this one. Maybe add it in an RFC in the repo. This is intended to be simple as possible.

I will fix README

**Comment by pedrobaeza** [comment]
I think it can be in the ROADMAP of this module, as it's totally related, but as you prefer.

**Comment by rousseldenis** [comment]
@pedrobaeza README and __manifest__ fixed. And yes I'd prefer (maybe add two modules in sale and stock repos ?)

**Review by pedrobaeza — APPROVED** [review]

**Comment by rousseldenis** [comment]
Rebased

**Comment by rafaelbn** [comment]
Hi @rousseldenis , runbot is :red_circle:

**Comment by rousseldenis** [comment]
@rafaelbn Yes. That's the description key. I'll fix it soon

**Comment by rousseldenis** [comment]
@rafaelbn All green!

**Comment by rafaelbn** [comment]
Hi @rousseldenis , I see the barcode field duplicated in my test in runbot

(Do you know that Odoo v11 already has `barcode` field in product.packaging?) :smile:

**Comment by rousseldenis** [comment]
Arf. I think I saw it but I'm not in v11 for now. That's Odoo back and forth strategy :smile:

**Comment by rafaelbn** [comment]
About the duplicated field barcode , is it expected en v10? if so :+1:

**Comment by rafaelbn** [comment]
Could you please add a note in README to be aware the next one when migrating this module to v11 please?

**Comment by rousseldenis** [comment]
@rafaelbn No need to migrate this to 11. The field exists...

**Review by rafaelbn — APPROVED** [review]

**Pull request merged**